docs/concept-workflows.md

```diff
@@ -60,70 +240,45 @@ public class SubscriptionWorkflowImpl implements SubscriptionWorkflow {
 
-To distinguish multiple runs of a Workflow with the same Workflow Id, Temporal identifies a Workflow with two Ids: `Workflow Id` and `Run Id`. `Run Id` is a service-assigned UUID. To be precise, any Workflow is uniquely identified by a triple: `Namespace`, `Workflow Id` and `Run Id`.
+The Temporal Server has practically no scalability limits on the number of open Workflow instances, so this code can be used over and over even if your application has hundreds of millions of customers.
 
-## Child Workflow
+## FAQ
 
-A Workflow can execute other Workflows as `child Workflows`. A child Workflow completion or failure is reported to its parent.
+**Is there a limit to length of time a Workflows can run?**
 
-Some reasons to use child Workflows are:
+Workflows intended to run indefinitely should be written with some care.
+Temporal stores the complete event history for the entire life of a Workflow Execution.
+There is a maximum limit of 50,000 events that is enforced by the Server, and you should try to avoid getting close to this limit; The Temporal Server puts out a warning at every 10,000 events.
 
-- A child Workflow can be hosted by a separate set of workers which don't contain the parent Workflow code. So it would act as a separate service that can be invoked from multiple other Workflows.
-- A single Workflow has a limited size. For example, it cannot execute 100k Activities. Child Workflows can be used to partition the problem into smaller chunks. One parent with 1000 children each executing 1000 Activities is 1 million executed Activities.
-- A child Workflow can be used to manage some resource using its Id to guarantee uniqueness. For example, a Workflow that manages host upgrades can have a child Workflow per host (host name being a Workflow Id) and use them to ensure that all operations on the host are serialized.
-- A child Workflow can be used to execute some periodic logic without blowing up the parent history size. When a parent starts a child, it executes periodic logic calling that continues as many times as needed, then completes. From the parent point of view, it is just a single child Workflow invocation.
+The idiomatic way to handle indefinitely running Workflows is to use the "Continue-as-new" feature, which is available in all SDKs.
+For example a reasonable cutoff point might be once a day for high volume Workflows.
 
-The main limitation of a child Workflow versus collocating all the application logic in a single Workflow is lack of the shared state. Parent and child can communicate only through asynchronous signals. But if there is a tight coupling between them, it might be simpler to use a single Workflow and just rely on a shared object state.
+The "Continue-as-new" feature completes the current Workflow execution and start a new execution with the same Workflow Id, passing it the appropriate parameters for it to continue.
```

Comment:
a starts a new execution with the same workflowId (and different runId) **atomically**.